src/components/profile/EditProfile.js

```diff
@@ -213,10 +213,10 @@ const getStylesFromProps = ({ theme }) => {
   return {
     animatedSaveButton: {
       position: 'absolute',
-      width: 120,
-      height: 60,
-      top: 0,
-      right: -24,
+      width: isBrowser ? 110 : 100,
```

Comment:
1. what if we'll reduce browser window's size ? Or will enable responsive mode with small device emulated
2. what if we're running on tablet with retina ? it's resolution will be the same as on the desktop

i think better to calculate button width relatively, as some percent from viewport / header etc

Reply:
fixed